content/changelog/2024-04-11-clever-tools-3.5.2.md

```diff
@@ -15,7 +15,7 @@ description: Lots of new features included!
 excludeSearch: true
 ---
 
-Clever Tools 3.5 is now available, and this branch brings lots of new important features. First, `clever curl` is now public and listed as an official command. It helps you to send cURL requests with the auth context of your Clever Tools configuration. Thus, you can use Clever Cloud [API v2 or v4](/api/).
+Clever Tools 3.5 is now available, and this branch brings lots of new important features. First, `clever curl` is now public and listed as an official command. It helps you to send cURL requests with the auth context of your Clever Tools configuration. Thus, you can use Clever Cloud [API v2 or v4](/api/)..
```

Comment:
Why two periods, here?

Reply:
It's a revert PR. So it removes fixes from the initial PR.

Reply:
```suggestion
Clever Tools 3.5 is now available, and this branch brings lots of new important features. First, `clever curl` is now public and listed as an official command. It helps you to send cURL requests with the auth context of your Clever Tools configuration. Thus, you can use Clever Cloud [API v2 or v4](/api/).
```